CHANGES.md

```diff
@@ -51,6 +51,8 @@ To be released.
     `Address`.  [[#369]]
  -  Removed unnecessary mutex in `Swarm<T>` to avoid continuous delays in peer
     registration in some situations.  [[#375]]
+ -  Fixed a bug that `TurnClient` had thrown `SocketException` saying
+    “A connect request was made on an already connected socket”.  [[#376]]
```

Comment:
It's both a human-readable message and a string value of an exception object at at time, so quite confusing to me as well, but I feel more natural to treat it as a code string to me.  Not a strong opinion though.

```suggestion
    `A connect request was made on an already connected socket`.  [[#376]]
```